### DISCUSSION TITLE
Ch.1 05. [실습] kubectl

### DISCUSSION THREAD

**Discussion by sooho410** [yoonjeong-kwon/fastcampus-kubernetes#22] [Q&A] [answered]
해당 실습 진행하려고 하는데요, 
vscode로 진행합니다.
kubernetes extension은 설치가 되어있는 상태인데 실습 진행하는 방법을 알려주시면 감사하겠습니다.

![image](https://user-images.githubusercontent.com/77769466/233101983-25790869-a9e6-4374-b036-e3c28158154c.png)

**Comment by yoonjeong-kwon** [comment, marked as answer]
@sooho410 님, 안녕하세요.
빨간색 에러 메시 내용으로 보아 kubectl CLI가 설치되지 않은 상태입니다.
Chapter 01 - 실습3부터 따라하시면 실습을 진행하실 수 있습니다.
Chapter 01 - 실습3 강의안에 슬라이드 31을 참고해주세요.
